starter-code/bin/seeds.js

```diff
@@ -1 +1,35 @@
 // Iteration #1
```

Comment:
Were you able to run this file?
If not, in `starter-code` you should do `node bin/seeds.js` to give you some dummy data to play around with in the server.